packages/react-data-grid/src/Grid.js

```diff
@@ -133,9 +133,17 @@ class Grid extends React.Component {
     this.viewport = viewport;
   };
 
+  setViewportContainerRef = (viewportContainer) => {
+    this.viewPortContainer = viewportContainer;
+  };
+
+  setEmptyViewRef = (emptyView) => {
+    this.emptyView = emptyView;
+  };
+
   render() {
-    let headerRows = this.props.headerRows || [{ref: (node) => this.row = node}];
-    let EmptyRowsView = this.props.emptyRowsView;
+    const { headerRows } = this.props;
```

Comment:
This will never be undefined.
https://github.com/adazzle/react-data-grid/blob/master/packages/react-data-grid/src/ReactDataGrid.js#L560
https://github.com/adazzle/react-data-grid/blob/master/packages/react-data-grid/src/ReactDataGrid.js#L700